---a/net/minecraft/util/ReportedException.java
+++b/net/minecraft/util/ReportedException.java
@@ -2,29 +2,25 @@
 
 import net.minecraft.crash.CrashReport;
 
-public class ReportedException extends RuntimeException
-{
+public class ReportedException extends RuntimeException {
+
     private final CrashReport theReportedExceptionCrashReport;
     private static final String __OBFID = "CL_00001579";
 
-    public ReportedException(CrashReport report)
-    {
+    public ReportedException(CrashReport report) {
         this.theReportedExceptionCrashReport = report;
     }
 
-    public CrashReport getCrashReport()
-    {
+    public CrashReport getCrashReport() {
         return this.theReportedExceptionCrashReport;
     }
 
-    public Throwable getCause()
-    {
+    public Throwable getCause() {
         return this.theReportedExceptionCrashReport.getCrashCause();
     }
 
-    public String getMessage()
-    {
+    public String getMessage() {
         return this.theReportedExceptionCrashReport.getDescription();
     }
-}
+}
